lib/fennec/evaluator/channel_data.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Fennec.Evaluator.ChannelData do
+  @moduledoc false
+
+  alias Fennec.TURN
+  alias Fennec.TURN.Channel
+  alias Jerboa.ChannelData
+
+  require Logger
+
+  @spec service(ChannelData.t, TURN.t) :: TURN.t
+  def service(channel_data, turn_state) do
+    case TURN.has_channel(turn_state, channel_data.channel_number) do
+      {:ok, turn_state, channel} ->
+        send(turn_state, channel, channel_data)
+        turn_state
+      {:error, turn_state} ->
+        Logger.debug fn ->
+          "Dropping data sent over channel ##{channel_data.channel_number}. " <>
```

Comment:
Why double `#`?

Reply:
I guess that it's just for prettier result, but I wanted to make sure I'm not missing something.

Reply:
Yup, that's it